doc/upgrade.rst

```diff
@@ -105,6 +105,8 @@ Then, modify postfix's configuration as follows::
 
 Replace ``<driver>`` and ``<path>`` by your values.
 
+If transport_maps contains sql-relaydomains-transport.cf, remove it.
```

Comment:
This line should appear in the dedicated instructions of the next major release (1.10.0 I guess).